www/src/components/showcase-details.js

```diff
@@ -0,0 +1,532 @@
+    color: colors.gray.calm,
+    fontFamily: options.headerFontFamily.join(`,`),
+    fontWeight: `normal`,
+  },
+  prevNextPermalinkImage: {
+    marginBottom: 0,
+    marginTop: rhythm(options.blockMarginBottom),
+  },
+  prevNextPermalinkTitle: {
+    color: colors.gatsby,
+    display: `block`,
+    position: `relative`,
+  },
+  prevNextPermalinkContainer: {
+    width: `50%`,
+  },
+  truncate: {
+    whiteSpace: `nowrap`,
+    overflow: `hidden`,
+    textOverflow: `ellipsis`,
+    display: `block`,
+    width: `100%`,
+  },
+  prevNextPermalinkArrow: {
+    color: colors.lilac,
+    marginRight: 4,
+    verticalAlign: `sub`,
+    position: `absolute`,
+    left: `-${rhythm(3 / 4)}`,
+    top: `50%`,
+    transform: `translateY(-50%)`,
+  },
+  prevNextPermalinkMeta: {
+    marginLeft: rhythm(6 / 4),
+    display: `flex`,
+    flexDirection: `row`,
+    justifyContent: `flex-end`,
+  },
+  prevNextPermalinkMetaInner: {
+    flexBasis: 540,
+    flexGrow: 0,
+    flexShrink: 1,
+    minWidth: 0,
+  },
+}
+
+const cleanUrl = mainUrl => {
+  const parsed = url.parse(mainUrl)
+  let path = parsed.pathname
+  if (path[path.length - 1] === `/`) path = path.slice(0, path.length - 1)
+  return parsed.hostname + path
+}
+
+export default ({ parent, data, isModal, categories }) => (
+  <StaticQuery
+    query={graphql`
+      query allSitesYamlTemplateShowcase {
+        allSitesYaml(filter: { fields: { slug: { ne: null } } }) {
+          edges {
+            node {
+              id
+              url
+              title
+              fields {
+                slug
+              }
+              childScreenshot {
+                screenshotFile {
+                  childImageSharp {
+                    resize(width: 200, height: 200) {
+                      src
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    `}
+    render={staticData => {
+      const allSitesYaml = staticData.allSitesYaml
+      const nextSite = parent.getNext(allSitesYaml)
+      const previousSite = parent.getPrevious(allSitesYaml)
+
+      return (
+        <Layout
+          location={parent.props.location}
+          isModal={isModal}
+          modalBackgroundPath="/showcase"
+          modalNext={() => parent.next(allSitesYaml)}
+          modalPrevious={() => parent.previous(allSitesYaml)}
+          modalNextLink={
+            <Link
+              to={{
+                pathname: nextSite.fields.slug,
+                state: {
+                  isModal: true,
+                },
+              }}
+              css={{
+                display: `block`,
+                position: `fixed`,
+                top: `150px`,
+                transform: `translateX(750px)`,
+              }}
+            >
+              <div
+                css={{
+                  margin: `25px`,
+                }}
+              >
+                <Img
+                  key={nextSite.id}
```

Comment:
This is what was necessary to stop the old images from showing temporarily

Reply:
i have a commit with this fix as well, will just scrap it